packages/light.js/src/rpc/eth.ts

```diff
@@ -45,6 +45,22 @@ export function balanceOf$ (address: Address, options?: RpcObservableOptions) {
   })(options)(address);
 }
 
+/**
+ * Get the transaction count of a given account. Calls `eth_getTransactionCount`
+ *
+ * @param address - Address of the account whose transaction count we want to query.
+ * @param options - Options to pass to {@link RpcObservableOptions}.
+ * @return - An Observable containing the transaction count.
+ */
+export function transactionCountOf$(address: Address, options?: RpcObservableOptions) {
+  return createRpc$<any, BigNumber | Symbol>({
+    calls: ['eth_getTransactionCount'],
+    frequency: [frequency.onEveryBlock$, frequency.onStartup$],
+    name: 'transactionCountOf$',
+    pipes: api => [switchMapPromise(() => api.eth.getTransactionCount(address).then(Number))]
```

Comment:
What is `then(Number)`?

Reply:
use the Number constructor to convert the value to a number
otherwise it's an object/string; I thought that it would make sense if transactionCountOf$ returns a number

Reply:
nice. So in this case the TS def should be `createRpc$<any, number | Symbol>`?

Reply:
ah yes, you're right!